src/App.css

```diff
@@ -27,6 +27,8 @@
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
+  position: sticky;
```

Comment:
it's better to stick the header element instead of the logo, to avoid the transparent issue.

Reply:
@akctba the header element is sticky not the logo, the app-header is `position: sticky;` and the logo - .wobble is not positioned sticky. Re check the code it already is what you want it to be. :)